improver/metadata/amend.py

```diff
@@ -30,25 +30,26 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Module containing utilities for modifying cube metadata"""
 from datetime import datetime
+from typing import Dict
 
 from dateutil import tz
+from iris.cube import Cube
 
 from improver.metadata.constants.mo_attributes import (
     GRID_ID_LOOKUP,
     MOSG_GRID_DEFINITION,
 )
 
 
-def update_stage_v110_metadata(cube):
+def update_stage_v110_metadata(cube: Cube):
     """Translates attributes relating to the grid_id attribute from StaGE
     version 1.1.0 to later StaGE versions.
     Cubes that have no "grid_id" attribute are not recognised as v1.1.0 and
     are ignored.
 
     Args:
-        cube (iris.cube.Cube):
+        cube:
             Cube to modify attributes in (modified in place)
-
     Returns:
         None
```

Comment:
We should delete this returns section whilst we are about it.

Reply:
Done